Source/Core/Common/SymbolDB.cpp

```diff
@@ -39,6 +39,45 @@ void SymbolDB::Index()
 	}
 }
 
+void SymbolDB::PopulateSymbolComments()
+{
+	std::string filename = File::GetUserPath(D_MAPS_IDX) + SConfig::GetInstance().m_strUniqueID + "_comments.txt";
+	if (!File::Exists(filename))
+		return;
+	std::fstream line_parse;
+	line_parse.open(filename);
+	std::string line;
+	u32 address;
+	while(!(line_parse>>std::ws).eof())
+	{
+		std::getline(line_parse, line);
+		if (line.empty())
+			continue;
+		AsciiToHex(line.substr(0, 8),address);
+		std::string comment = line.substr(9, std::string::npos);
+		if (address < 0x80000010)
```

Comment:
There should be a comment indicating what this constant is, or store it to a constexpr variable.